contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -780,7 +780,7 @@ contract VaultHub is PausableUntilWithRoles {
         VaultConnection storage connection = _checkConnectionAndOwner(_vault);
 
         connection.isBeaconDepositsManuallyPaused = true;
-        IStakingVault(_vault).pauseBeaconChainDeposits();
+        _pauseBeaconChainDepositsIfNotAlready(IStakingVault(_vault));
```

Comment:
we change semantics here :warning: 

probably shall revisit this and add an event for the flag itself